packages/edit-site/src/components/code-editor/style.scss

```diff
@@ -1,7 +1,7 @@
 .edit-site-code-editor {
 	position: relative;
 	width: 100%;
-	min-height: 100%;
```

Comment:
This keeps the background filllng the screen.
<img width="910" alt="Screenshot 2024-05-08 at 14 15 50" src="https://github.com/WordPress/gutenberg/assets/275961/27baaba0-5ce1-4b6b-9fa3-8f8b1cc421f2">